playgrounds/react/examples/list-example.tsx

```diff
@@ -15,8 +15,11 @@ interface IListItemProps {
 }
 
 const items: IListItemProps[] = [
+  // @ts-expect-error
```

Comment:
Why so?

Reply:
Looks like the original author just reused the existing interface instead of creating another one. In any case, fixed. I also this comment from similar instances.